src/command/output.rs

```diff
@@ -324,6 +342,15 @@ impl RoverOutput {
             RoverOutput::ErrorExplanation(explanation_markdown) => {
                 json!({ "explanation_markdown": explanation_markdown })
             }
+            RoverOutput::ReadmeFetchResponse(readme_response) => {
+                json!({ "readme": readme_response })
```

Comment:
`last_updated_at` should likely be included here instead of in the publish output since after publishing, `last_updated_at` will be "now", whereas it would generally be further in the past if it was just fetched.

Reply:
added!